packages/handlersjs-http/lib/handlers/routed-http-request.handler.ts

```diff
@@ -120,6 +120,7 @@ export class RoutedHttpRequestHandler implements HttpHandler {
       const requestWithParams = Object.assign(request, { parameters });
       const newContext = { request: requestWithParams, route: matchingRouteWithOperation.route };
       const preResponseHandler = matchingRouteWithOperation.controller.preResponseHandler;
+      const matchingOperation = matchingRouteWithOperation.route.operations.find((op) => op.method === request.method);
```

Comment:
What if you make line 105 the following:
```
const matchingRouteWithOperation = matchingRoutes
  .map((r) => ({ ...r, operation: r.route.operations.find((op) => op.method === request.method) }))
  .find((r) => r.operation);
```

Reply:
That would create a type `HttpHandlerRoute && { operation: HttpHandlerOperation }` if im not mistaken.
I don't dislike the solution but I think it might be confusting to read over that later on.
Would love to hear @lem-onade 's opinion on this.

Reply:
Ah that does make sense. I thought `matchingRouteWithOperation` had a type of `HttpHandlerRoute` and we would make it overly complex by doing this. But it was already an object containing a `HttpHandlerRoute` and a `HttpHandlerController`.
Past me got me good with the variable nameing and I should have double checked the type.
That stings :)

Reply:
Yeah variable naming could be a lot better there 😆 largely my bad!